ci/test/test_ci.py

```diff
@@ -23,17 +24,6 @@
     user2_token = f.read()
 
 
-def wait_for_hello():
-    wait_cmd = f'python3 wait-for.py 900 {namespace} Service --location k8s hello'
```

Comment:
You can now remove wait-for.py from the test image (yay!)

Reply:
erm, so I realized the commented out test here still uses this method, so I removed everything here that isn't actually relevant to a test. Are you OK with that?